### PR TITLE
add dump_sim_mode to Standalone HF_ICECLASS

### PR DESCRIPTION
**Pull request by iconsec**
Add a mode to the HF_ICECLASS standalone mode.

This mode:
1. Reads 1 iClass tag (in the same fashion as the read & dump mode) to iceclass-temp.bin in mem.
2. Sims that iceclass-temp.bin file until the mode has been exited.

This mode allows the reading/dumping/simming of a iClass tag in one mode.

### CODE REVIEW

File: armsrc/Standalone/hf_iceclass.c [hunk 1]
```diff
@@ -37,22 +37,28 @@
...
 // Select which standalone function to be active.
-// 4 possiblities.  Uncomment the one you wanna use.
+// 5 possiblities.  Uncomment the one you wanna use.
 
-#define ICE_USE               ICE_STATE_FULLSIM
+//#define ICE_USE               ICE_STATE_FULLSIM
 //#define ICE_USE               ICE_STATE_ATTACK
 //#define ICE_USE               ICE_STATE_READER
 //#define ICE_USE               ICE_STATE_CONFIGCARD
+#define ICE_USE               ICE_STATE_DUMP_SIM
```
Comment (iceman1001) [inline armsrc/Standalone/hf_iceclass.c:52]: The default mode is FULLSIM

**Review by iceman1001 — CHANGES_REQUESTED** [review]
Minor change need to revert back old default mode.

The save file to spiffs could also be refactored to take a filename instead of adding more functions

**Comment by iceman1001** [comment]
Nice

File: armsrc/Standalone/hf_iceclass.c (hunk 1 above)
Comment (iconsec) [reply armsrc/Standalone/hf_iceclass.c:52]: got it; easy fix.

**Review by iceman1001 — APPROVED** [review]

**Comment by iceman1001** [comment]
Nice!
Don't forget to add a line in changelog.md

**Pull request merged**